### ISSUE TITLE
About voc_dataset.py

### ISSUE THREAD

**Issue by wulele2** [ChingHo97/FCOS-PyTorch-37.2AP#1]
Hi, If an image has no objects, the corresponding tag file is empty .In the preprocess_img_boxes function, boxes=array([]). In the later collect_fn function, an error is reported. Because it's an empty array. How to modify this bug? Or how to tag background images, including classes and boxes. I look forward to hearing from you soon.

**Comment by ChingHo97** [comment]
@wulele2  Thank you for this bug, In voc dataset I didn't meet this problem and ignored it, you can in __init__  remove the no objects' img_id by judging the correspond xml. I will fix this bug soon.

**Comment by wulele2** [comment]
Thanks for your reply! Half of my tasks now include only background images, and I don't know what to do with them. According to voc_dataset. Py, just read in the picture, the tag is empty. Could you give me some advice? Thank you very much.

**Comment by wulele2** [comment]
I tried tagging images that only included backgrounds: classes=[0],boxes=[0,0,0,0]. Do you think that would work?

**Comment by ChingHo97** [comment]
@wulele2 if you would like train these images with no objects, I think you can use this and in the loss func, for every image in a loop, remove the class id = 0, if the gt num is 0, you can only compute it's focal loss. You can refer to https://github.com/yhenon/pytorch-retinanet/blob/master/retinanet/losses.py .

**Comment by wulele2** [comment]
really appreciate your help！
